.gitignore

```diff
@@ -2,4 +2,5 @@
 *.tz
 __pycache__/
 .env
-.vscode
\ No newline at end of file
+.vscode
+.ligo
```

Comment:
I don't really mind here, but don't forget you can also put stuff in your user gitignore instead of changing this one.